composer.lock

```diff
@@ -1695,33 +1685,29 @@
         },
         {
             "name": "paragonie/random_compat",
-            "version": "v2.0.17",
+            "version": "v9.99.99",
```

Comment:
What happened here?

Reply:
Many dependencies we have actually reference this PHP 7 compatibility package that we definitely don't need anymore since we now support PHP 7 onwards. So the creator of the package made a special version that is pulled on PHP 7 only that does... nothing, since `random_bytes` is natively available, but it satisfies the dependencies of our dependencies.